app/src/main/java/com/gdgevents/gdgeventsapp/ui/theme/Color.kt

```diff
@@ -1,11 +1,153 @@
+val inverseOnSurfaceDark = Color(0xFF313030)
+val inversePrimaryDark = Color(0xFF005BC0)
+val surfaceDimDark = Color(0xFF141313)
+val surfaceBrightDark = Color(0xFF3A3939)
+val surfaceContainerLowestDark = Color(0xFF0E0E0E)
+val surfaceContainerLowDark = Color(0xFF1C1B1C)
+val surfaceContainerDark = Color(0xFF201F20)
+val surfaceContainerHighDark = Color(0xFF2B2A2A)
+val surfaceContainerHighestDark = Color(0xFF353435)
+
+val primaryDarkMediumContrast = Color(0xFFB3CBFF)
+val onPrimaryDarkMediumContrast = Color(0xFF001537)
+val primaryContainerDarkMediumContrast = Color(0xFF4A8EFF)
+val onPrimaryContainerDarkMediumContrast = Color(0xFF000000)
+val secondaryDarkMediumContrast = Color(0xFFB3CBFF)
+val onSecondaryDarkMediumContrast = Color(0xFF001537)
+val secondaryContainerDarkMediumContrast = Color(0xFF4A8EFF)
+val onSecondaryContainerDarkMediumContrast = Color(0xFF000000)
+val tertiaryDarkMediumContrast = Color(0xFFFFFFFF)
+val onTertiaryDarkMediumContrast = Color(0xFF1E333B)
+val tertiaryContainerDarkMediumContrast = Color(0xFFC1D8E2)
+val onTertiaryContainerDarkMediumContrast = Color(0xFF0C2229)
+val errorDarkMediumContrast = Color(0xFFFFBAB1)
+val onErrorDarkMediumContrast = Color(0xFF370001)
+val errorContainerDarkMediumContrast = Color(0xFFFF5449)
+val onErrorContainerDarkMediumContrast = Color(0xFF000000)
+val backgroundDarkMediumContrast = Color(0xFF10131A)
+val onBackgroundDarkMediumContrast = Color(0xFFE0E2EC)
+val surfaceDarkMediumContrast = Color(0xFF141313)
+val onSurfaceDarkMediumContrast = Color(0xFFFEFAF9)
+val surfaceVariantDarkMediumContrast = Color(0xFF444748)
+val onSurfaceVariantDarkMediumContrast = Color(0xFFC8CBCC)
+val outlineDarkMediumContrast = Color(0xFFA0A3A4)
+val outlineVariantDarkMediumContrast = Color(0xFF808484)
+val scrimDarkMediumContrast = Color(0xFF000000)
+val inverseSurfaceDarkMediumContrast = Color(0xFFE5E2E1)
+val inverseOnSurfaceDarkMediumContrast = Color(0xFF2B2A2A)
+val inversePrimaryDarkMediumContrast = Color(0xFF004595)
+val surfaceDimDarkMediumContrast = Color(0xFF141313)
+val surfaceBrightDarkMediumContrast = Color(0xFF3A3939)
+val surfaceContainerLowestDarkMediumContrast = Color(0xFF0E0E0E)
+val surfaceContainerLowDarkMediumContrast = Color(0xFF1C1B1C)
+val surfaceContainerDarkMediumContrast = Color(0xFF201F20)
+val surfaceContainerHighDarkMediumContrast = Color(0xFF2B2A2A)
+val surfaceContainerHighestDarkMediumContrast = Color(0xFF353435)
+
+val primaryDarkHighContrast = Color(0xFFFBFAFF)
+val onPrimaryDarkHighContrast = Color(0xFF000000)
+val primaryContainerDarkHighContrast = Color(0xFFB3CBFF)
+val onPrimaryContainerDarkHighContrast = Color(0xFF000000)
+val secondaryDarkHighContrast = Color(0xFFFBFAFF)
+val onSecondaryDarkHighContrast = Color(0xFF000000)
+val secondaryContainerDarkHighContrast = Color(0xFFB3CBFF)
+val onSecondaryContainerDarkHighContrast = Color(0xFF000000)
+val tertiaryDarkHighContrast = Color(0xFFFFFFFF)
+val onTertiaryDarkHighContrast = Color(0xFF000000)
```

Comment:
unused Medium Contrast & High Contrast colors , please clean the code before submitting it